libraries/cyclestreets-core/src/main/java/net/cyclestreets/api/Map.kt

```diff
@@ -0,0 +1,8 @@
+package net.cyclestreets.api
+
+data class Map(
+        val id: String,
```

Comment:
picky, but 8 characters indentation seems excessive.

Also I wonder if we would be better off naming this class differently - e.g. `MapPack` - so we don't confuse it with the native `Map` class in Kotlin and/or Java.

Reply:
I now see it can't be `MapPack` as that would conflict.
What about `MapReference` or something similar; as effectively it acts as a reference to a downloadable map pack?

Reply:
I started with MapPack, realised, changed to Map as a holding name, now considering DownloadableMap